src/main/java/io/fabric8/jenkins/openshiftsync/JenkinsUtils.java

```diff
@@ -119,17 +120,42 @@ public static String getRootUrl() {
 		return root;
 	}
 
-	public static boolean verifyEnvVars(Map<String, ParameterDefinition> paramMap, WorkflowJob workflowJob) {
+	public static boolean verifyEnvVars(Map<String, ParameterDefinition> paramMap, WorkflowJob workflowJob) throws AbortException {
         if (paramMap != null) {
             String fullName = workflowJob.getFullName();
-            WorkflowJob job = Jenkins.getActiveInstance()
-                    .getItemByFullName(fullName,
-                            WorkflowJob.class);
-            if (job == null) {
+            WorkflowJob job = null;
+            long now = System.currentTimeMillis();
+            boolean anyErrors = false;
+            while (System.currentTimeMillis() - 5000 > now) {
+                job = Jenkins.getActiveInstance()
+                        .getItemByFullName(fullName,
+                                WorkflowJob.class);
+                
+                if (job != null) {
+                    if (anyErrors)
```

Comment:
Any reason for logging found message only in retry case `if(anyErrors)`

Reply:
I did not want to flood the log with error messages when things are down.

Reply:
Sorry, meant add logs in the normal case

Reply:
acknowledged @gabemontero